src/windows/CameraProxy.js

```diff
@@ -758,16 +759,31 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
         }, successCallback, errorCallback);
     };
 
-    // add and delete focus eventHandler to capture the focus back from cameraUI to app 
-    window.addEventListener("focus", savePhotoOnFocus);
-    cameraCaptureUI.captureFileAsync(WMCapture.CameraCaptureUIMode.photo).done(function(picture) {
+    // if windows phone 10, add and delete focus eventHandler to capture the focus back from cameraUI to app
+    if (navigator.appVersion.indexOf('Windows Phone 10.0') >= 0) { 
```

Comment:
Is this the best/only way to check this? It will fail on future versions of Windows Phone if the version number changes to something like 10.1 or 11.0.

And what about non-phone Windows 10?